draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -737,6 +737,11 @@ these effects also ought to be evaluated.
 
 Another example use is concurrent multipath, where the transport protocol simultaneously
 schedules flows to aggregate the capacity across multiple paths.
+{{?RFC6356}} is an Experimental RFC specifying
+a concurrent multipath congestion control algorithm for TCP (MP-TCP).
+{{?RFC8041}} discusses use cases
+and operational experience in real networks.
```

Comment:
I don't apply this rule consistently, but this seems a little tutorial-mode.

Reply:
Fair comment, I agree, this text is tutorial. There was a request to include RFC8041, and that was the shortest text I could add. Since the reference provides very little to help identify how to specify or evaluate a CC, I'd be very happy to not add that sentence. Thoughts?

Reply:
I would delete it, but it's not a big deal if you'd prefer to keep it.